.github/CODEOWNERS

```diff
@@ -52,8 +52,8 @@ exporter/dynatraceexporter/                              @open-telemetry/collect
 exporter/elasticsearchexporter/                          @open-telemetry/collector-contrib-approvers @JaredTan95
 exporter/f5cloudexporter/                                @open-telemetry/collector-contrib-approvers @gramidt
 exporter/fileexporter/                                   @open-telemetry/collector-contrib-approvers @atingchen
-exporter/googlecloudexporter/                            @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @tbarker25 @damemi
-exporter/googlemanagedprometheusexporter/                @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @tbarker25 @damemi
+exporter/googlecloudexporter/                            @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @damemi @psx95 #emeritus: @tbarker25
+exporter/googlemanagedprometheusexporter/                @open-telemetry/collector-contrib-approvers @aabmass @dashpole @jsuereth @punya @damemi @psx95 #emeritus: @tbarker25
```

Comment:
Do we want to start adding the emeritus comment? This needs to be a policy we agree on with @open-telemetry/collector-contrib-approvers. We haven't done that so far.

Reply:
I approved this PR because I was requesting changes earlier, but this indeed made me tick a bit. I don't see value in listing folks as emeritus, but maybe it's worth celebrating them elsewhere on README.md.

Reply:
I don't think we should list emeritus in this file which we use as an up-to-date source of truth for our CI automation

Reply:
I agree. @damemi, can you please remove the comment?

Reply:
FYI - https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager

Reply:
> FYI - https://github.com/open-telemetry/community/blob/main/community-membership.md#emeritus-maintainerapprovertriager

I don't think code owners as used in the Collector quite fit into the maintainer/approver/triager roles, but still I think it's a good idea to have both a current owners and an emeritus list on each component's README. 

However, I think we should deal with this separately: we are not doing it now for any other component, and we definitely don't want to do it in the CODEOWNERS file, so for this PR I agree with @dmitryax and @TylerHelmuth that we should remove the comment.

Reply:
In the interest of keeping this scoped to the main goal, I updated this PR to just add @psx95 and not touch @tbarker25. The topic of emeritus approvers is broader than what I intended for this change. I'm sorry for the confusion, everyone.